extensions/src/main/java/org/pentaho/platform/web/http/api/resources/services/SchedulerService.java

```diff
@@ -85,8 +89,8 @@ public Job createJob( JobScheduleRequest scheduleRequest )
 
     // Used to determine if created by a RunInBackgroundCommand
     boolean runInBackground =
-        scheduleRequest.getSimpleJobTrigger() == null && scheduleRequest.getComplexJobTrigger() == null
-            && scheduleRequest.getCronJobTrigger() == null;
+      scheduleRequest.getSimpleJobTrigger() == null && scheduleRequest.getComplexJobTrigger() == null
+        && scheduleRequest.getCronJobTrigger() == null;
```

Comment:
I personally disagree, I thing it's helpful to log all stages, but we can discuss at team level. Yes, this si being done with disregard for which mode we're in (WN vs non-WN). As we discussed in grooming, it doesn't hurt to have in non-WN mode and may actually be more work to disable this functionality in non-WN mode.